cypress/e2e/end-end/automatedtestactivity_clamsim_ap_spec.js

```diff
@@ -0,0 +1,79 @@
+import { uid } from 'uid';
+import * as TeacherHelper from "../../support/helpers/teacherHelper";
+import {
+  BTN_ACTIVITY_RUN,
+  joinClass,
+  getLinkGenerateReport
+} from "../../support/elements/student_home_page_elements";
+import * as adminHelper from "../../support/helpers/adminHelper";
+import * as apRuntimeHelper from '../../support/helpers/apRuntimeHelper';
+import * as clamSimHelper from '../../support/helpers/clamSimHelper';
+import * as C from "../../support/constants";
+import {ClamSimElements as e} from "../../support/elements/clam_sim_elements";
+import {experimentConfigs} from "../../support/testdata/testdata_automatedtestactivity_clamsim_ap";
+
+const CLASS_WORD = C.CLASS_WORD;
+const CLASS_NAME = 'Class '+ CLASS_WORD;
+const ASSIGNMENT_NAME = 'Automation Clam Sim Activity';
+const STUDENTS = ["STUDENT4"];
+
+context("Verify Student Acitivty Player Activity Work Flow", () => {
+
```

Comment:
Shouldn't almost the entire file, from here to just above the last line, be indented by 2 additional spaces? (Sorry to harp on indenting so much, but it really helps me (& others, I believe) to see the structure of the file, and what is inside what. This indenting makes it appear that "before" and "clearCookies", etc., are outside the "context", which is not actually true.)

Reply:
Usually for the import and variable declarations we are not providing the indented spaces. So, I left those as is matching the other test files.